website/blog/2019-03-04-tutorial.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Tutorial: Estimating health impacts of the Volkswagen emissions scandal
-author: Xinyu Zhang
-authorURL: xyzhang8@uw.edu
+author: Xinyu Zhang xyzhang8@uw.edu
```

Comment:
Please change to `Xinyu Zhang (xyzhang8@uw.edu)`. Or, unless you really want people to be emailing you about this, you could remove the email address altogether.

Reply:
Thank you! I have submitted another change.